Gemfile

```diff
@@ -18,7 +20,7 @@ group :development, :test do
   gem "sqlite3" # Remove this when you choose a production database
   gem "factory_girl_rails", "~> 4.7"
   gem "timecop"
-  gem "webmock", require: false
+  gem "webmock"
```

Comment:
For some reason we had to do that to make it run the `WebMock.disable_net_connect!(allow_localhost: true)` line properly.